migrations/migration1/migration1-up.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE public.languages (
+    id SERIAL PRIMARY KEY NOT NULL,
+    name character varying(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE public.locations (
+    id SERIAL PRIMARY KEY NOT NULL,
+    name character varying(255) UNIQUE NOT NULL
+);
+
+CREATE TABLE public.users (
+    id SERIAL PRIMARY KEY NOT NULL,
+    name character varying(255) UNIQUE NOT NULL,
```

Comment:
Is there a more compact way of specifying this, optimally on the same line of `location_id`?

Reply:
Certainly, we can drop the verbose ```FOREIGN_KEY``` syntax and just add ```REFERENCES public.locations(id)``` on the same line. The operation can also be performed on the ```user_languages``` table